No output when syntax errors in Go file

I run the command in a dir where there's a go file which looks like this (the curly brace on a new line):

```
func main() 
{
	fmt.Println("hello world")
}
```
There's zero output, however I would expect it to forward the formatter error message to the console. Same with any other file contents not matching go, but having a .go extension.

I can reproduce the issue by changing the repo:
```
$ git diff
diff --git a/examples/go/main.go b/examples/go/main.go
index c048119..6e81588 100644
--- a/examples/go/main.go
+++ b/examples/go/main.go
@@ -2,6 +2,8 @@ package main
 
 import "fmt"
 
-func main() {
+func main()
+
+{
        fmt.Println("hello world")
 }
```

```console
$ treefmt -C examples/go/

$ echo $?
1
$ gofmt examples/go/
examples/go/main.go:7:1: unexpected semicolon or newline before {
```

Treefmt should display the error

This changed in the last release. With treefmt v0.4.0:
```console
$ treefmt -C examples/go/
[INFO]: Error using formatter #go:
• [STDOUT]:

• [STDERR]:
/home/zimbatm/go/src/github.com/numtide/treefmt/examples/go/main.go:7:1: unexpected semicolon or newline before {

[ERR]: #go's formatter failed: exit status 2
```

--- a/src/formatter.rs
+++ b/src/formatter.rs
@@ -1,13 +1,15 @@
 //! Utilities for the formatters themselves.
-use crate::config::FmtConfig;
-use crate::{expand_exe, expand_if_path, expand_path};
+use std::{fmt, path::Path, path::PathBuf, process::Command};
+
 use anyhow::{anyhow, Result};
 use console::style;
 use globset::{GlobBuilder, GlobSet, GlobSetBuilder};
-use log::{debug, info};
+use log::{debug, warn};
 use serde::de::{self, Visitor};
 use serde::{Deserialize, Deserializer, Serialize, Serializer};
-use std::{fmt, path::Path, path::PathBuf, process::Command};
+
+use crate::config::FmtConfig;
+use crate::{expand_exe, expand_if_path, expand_path};
 
 /// newtype for the formatter name
 #[derive(Debug, Default, Clone, PartialEq, Eq, PartialOrd, Ord)]
@@ -102,7 +104,7 @@ impl Formatter {
         match cmd.output() {
             Ok(out) => {
                 if !out.status.success() {
-                    info!(
+                    warn!(
                         "Error using formatter {}:\n{stdout}:\n{}\n{stderr}:\n{}",
                         self.name,
                         String::from_utf8_lossy(&out.stdout),

--- a/src/main.rs
+++ b/src/main.rs
@@ -18,7 +18,7 @@ fn run() -> anyhow::Result<()> {
         .format_timestamp(None)
         .format_target(false)
         .filter_level(match cli.verbosity {
-            0 => LevelFilter::Off,
+            0 => LevelFilter::Warn,
             1 => LevelFilter::Info,
             2 => LevelFilter::Debug,
             _ => LevelFilter::Trace,

